recipes/sphinx-autosummary-accessors/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sphinx-autosummary-accessors" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 808c41542fffdf5ead6e74a291b4294191adc78d03fc53b945bf7c5ece299d27
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - importlib-metadata
+    - packaging
+    - python >=3.6
+    - sphinx
+
+test:
+  imports:
+    - sphinx_autosummary_accessors
+  commands:
+    - pip check
```

Comment:
The error on windows is due to `pip check`, although it looks to maybe be spurious since `colorama` is only required for win32 (https://github.com/sphinx-doc/sphinx/blob/eddfa686bef5955335576cb2b1c91931ac825cb9/setup.py#L38-L40), and I think we're building on win64. I think since this is `noarch: python`, we can ignore this error since the recipe will only be built on linux anyways. 

@conda-forge/staged-recipes -- Any thoughts on this?

Reply:
Yeah, this is noarch so we can ignore